frappe/__init__.py

```diff
@@ -28,6 +28,8 @@
 from .utils.jinja import (get_jenv, get_template, render_template, get_email_from_template, get_jloader)
 from .utils.lazy_loader import lazy_import
 
+from frappe.query_builder import query_builder
```

Comment:
```suggestion
from frappe.query_builder import get_query_builder
```
---
Maybe we should change the namespaces for them too. Have `frappe.query_builder.functions` and `frappe.query_builder.builder` instead of the current ones.